rkt/pods.go

```diff
@@ -920,6 +921,36 @@ func (p *pod) getContainerPID1() (pid int, err error) {
 	}
 }
 
+// getCgroup returnes the cgroup of pod.
+// Note that that returned string is not a file path, i.e. it does not containes
+// '/sys/fs/cgroup' as a prefix.
+func (p *pod) getCgroup() (string, error) {
+	pid, err := p.getContainerPID1()
+	if err != nil {
+		return "", err
+	}
+
+	cgroupFile, err := os.Open(fmt.Sprintf("/proc/%d/cgroup", pid))
+	if err != nil {
+		return "", err
+	}
+	scanner := bufio.NewScanner(cgroupFile)
+	for scanner.Scan() {
+		// See man proc for the format of the cgroup files.
+		// Basically it's in the form of:
+		// [ID]:[subsystem]:[cgroup]
+		tuples := strings.Split(scanner.Text(), ":")
+
+		// Read the systemd cgroup.
+		// TODO(yifan): What if rkt runs on non-systemd system?
+		// Read the cgroup of 'ID == 1'?
+		if tuples[1] == "name=systemd" {
```

Comment:
This will need to be reworked with unified hierarchy: #1757, but we will see at that time